contributing/translations/index.rst

```diff
@@ -0,0 +1,90 @@
+Contributing Translations
+=========================
+
+Some Symfony Components include certain messages that must be translated to
+different languages. For example, if a user submits a form with a wrong value in
+a :doc:`TimezoneType </reference/forms/types/timezone>` field, Symfony shows by
+default the following error message: "This value is not a valid timezone."
+
+These messages are translated into tens of languages thanks to the Symfony
+community. Symfony adds new messages on a regular basis, so this is an ongoing
+translation process and you can help us providing the missing translations.
+
+How to Contribute a Translation
+-------------------------------
+
+Imagine that you can speak both English and Swedish and want to check if there's
+some missing Swedish translations to contribute them.
+
+**Step 1.** Translations are contributed to the oldest maintained branch of the
+Symfony repository. Visit the `Symfony Releases`_ page to find out which is the
+current oldest maintained branch.
+
+If you know Git well, clone the Symfony repository and check out that branch.
+Otherwise, visit https://github.com/symfony/symfony and switch to that branch.
```

Comment:
could link to https://symfony.com/doc/current/contributing/index.html instead wdyt?

Reply:
I don't follow you ... they need to go to GitHub website if they want to make these changes without Git or the console, right?

Reply:
i meant to point the document already existing to the contributing part 

`If you know Git well` ... maybe in the otherwise, you could point to this page
sometime translations are first issue and this page is worth reading prior, even with already good knowledge of Git

Reply:
Now I know what you mean. You are right! I've made some changes. Thanks!